src/Payments.Mvc/Controllers/ReportsController.cs

```diff
@@ -25,12 +23,14 @@ public IActionResult Index()
         }
 
         #region team reports
-        public IActionResult Activity(string team) {
+        public IActionResult Activity(string team)
+        {
             // TODO: add in date range filters
 
             var invoices = _dbContext.Invoices
+                .Include(i => i.Account)
                 .Where(i => i.Team.Slug == TeamSlug)
-                .Where(i => i.CreatedAt >= DateTime.Now.AddMonths(-3))
+                .Where(i => i.CreatedAt >= DateTime.Now.AddMonths(-12))
```

Comment:
UTC Now? Can't remember if that is needed here.

Reply:
well in azure they run in UTC so now and UTC now are the same.  But since we did manually set CreatedAt = UtcNow when creating the invoice you are right for simplicity i think i probably should change to utcnow